packages/instance-model/lib/model.js

```diff
@@ -256,6 +257,33 @@ const InstanceModel = AmpersandModel.extend(
       }
     },
 
+    /**
+     * Check if the namespace supports list search indexes.
+     * This is actually a feature of a cluster and not of a collection.
+     * As $listSearchIndexes aggregation runs against a namespace, we perform
+     * this check for a namespace and then save it on the model itself and then
+     * use that value throughout.
+     * 
+     * @param {{ ns: string, dataService: import('mongodb-data-service').DataService }} dataService
+     * @returns {Promise<boolean>}
+     */
+    async getIsSearchSupported({
+      ns,
+      dataService,
+      force = false
+    }) {
+      if (this.isSearchIndexesSupported === undefined || force) {
+        try {
+          const isSearchIndexesSupported = await dataService.isListSearchIndexesSupported(ns);
+          this.set({ isSearchIndexesSupported });
+        } catch (e) {
```

Comment:
Why this try/catch? isListSearchIndexesSupported's entire body is already a try/catch?